### PR TITLE
[Release] 4.3.1

### PR DESCRIPTION
**Pull request by damacus**
### Description

Release 4.3.1

### Issues Resolved

- #572 
- #571 
- #566 

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by Xorima** [comment]
Hey @damacus 

Could you reccy `dokken list` with the suites and os defined in: https://github.com/sous-chefs/java/blob/master/.github/workflows/ci.yml

Looks like whoever migrated us to github actions may have made some dumb mistakes 🦆

**Comment by damacus** [comment]
Mind if I do that on the next PR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 This file is used to list changes made in each version of the Java cookbook.
 
-## [Unreleased]
-
-### Added
+## 4.3.1 (2020-02-21)
```
Comment (tas50) [inline CHANGELOG.md]: This should at least be 4.4. if we're going to nuke all that old stuff if not 5.0

**Comment by JohnRoesler** [comment]
5 got released https://github.com/sous-chefs/java/pull/588

**Pull request closed without merging**